session/tlv.go

```diff
@@ -237,8 +237,8 @@ func DeserializeSession(r io.Reader) (*Session, error) {
 	session.Label = string(label)
 	session.State = State(state)
 	session.Type = Type(typ)
-	session.Expiry = time.Unix(int64(expiry), 0)
-	session.CreatedAt = time.Unix(int64(createdAt), 0)
+	session.Expiry = time.Unix(int64(expiry), 0).UTC()
+	session.CreatedAt = time.Unix(int64(createdAt), 0).UTC()
```

Comment:
Hmmm can this somehow lead to some discrepancy between current sessions that were created without UTC, which are now deserialised with UTC?

Reply:
did some testing just to make sure no issues.

Reply:
So I guess the worst case scenario would be that a user after upgrading has a very recently created session that's now deserialized with a timestamp that's a few hours in the future. Alternatively the same could occur for recently revoked sessions, that's seen as having been revoked in the future.

Other than that such scenarios would be pretty confusing (which I think would be fine), there's one edge case where that could lead to a recently created session being automatically revoked:
https://github.com/lightninglabs/lightning-terminal/blob/c0e85e0f243aad6360499d9b553beab4c7461f98/session_rpcserver.go#L442-L448

But given that it's such an edge case where the worst case scenario would jsut be that a session gets revoked, we deemed this to be an ok tradeoff offline.